client/state/staging-site/constants.ts

```diff
@@ -0,0 +1,10 @@
+export enum StagingSiteStatus {
+	UNSET = '',
+	NONE = 'none',
+	REVERTING = 'reverting',
+	INITIATE_TRANSFERRING = 'initiate transferring',
+	INITIATE_REVERTING = 'initiate reverting',
+	TRANSFERRING = 'transferring',
+	COMPLETE = 'complete',
+	REVERTED = 'reverted',
+}
```

Comment:
What do you think if we reuse the existing transfer status?

https://github.com/Automattic/wp-calypso/blob/6704bd781443cc4a60f10949844a573dc6dc20f8/client/state/automated-transfer/constants.ts#L1-L28

You are already using them on `client/state/automated-transfer/constants.ts:30`

Reply:
How do you think we should reuse that? Actually what it's happening now is a mapping from more to less states, in order to have better control of what's happening.
Thanks for your comment!

Reply:
@kozer, Thanks for mapping the existing states https://github.com/Automattic/wp-calypso/pull/83545/commits/b90185245e949c47222e489ea836e70272efb127

This allows us to differentiate states coming from the network and synthetic states, like `initiate transferring`.

All good on my side to merge 👌